File: Java/Kode Program/Petshop.java

public class Petshop
{

    private String id;
    private String namaProduk;
    private String kategoriProduk;
    private int hargaProduk;

    public Petshop() 
    {
        // Konstruktor default kosong
    }

    public void setId(String id) 
    {
        this.id = id;
    }

    public String getId() 
    {
        return id;
    }

    public void setNamaProduk(String namaProduk) 
    {
        this.namaProduk = namaProduk;
    }

    public String getNamaProduk() 
    {
        return namaProduk;
    }

    public void setKategoriProduk(String kategoriProduk) 
    {
        this.kategoriProduk = kategoriProduk;
    }

    public String getKategoriProduk() 
    {
        return kategoriProduk;
    }

    public void setHargaProduk(int hargaProduk) 
    {
        this.hargaProduk = hargaProduk;
    }

    public int getHargaProduk() 
    {
        return hargaProduk;
    }
}

File: PHP/Kode Program/Petshop.php
<?php
class Petshop {
    private $id;
    private $nama_produk;
    private $kategori_produk;
    private $harga_produk;

    public function __construct($id = "", $nama_produk = "", $kategori_produk = "", $harga_produk = 0) 
    {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->kategori_produk = $kategori_produk;
        $this->harga_produk = $harga_produk;
    }

    public function setId($id) 
    {
        $this->id = $id;
    }

    public function getId() 
    {
        return $this->id;
    }

    public function setNamaProduk($nama_produk) 
    {
        $this->nama_produk = $nama_produk;
    }

    public function getNamaProduk() 
    {
        return $this->nama_produk;
    }

    public function setKategoriProduk($kategori_produk) 
    {
        $this->kategori_produk = $kategori_produk;
    }

    public function getKategoriProduk() 
    {
        return $this->kategori_produk;
    }

    public function setHargaProduk($harga_produk) 
    {
        $this->harga_produk = $harga_produk;
    }

    public function getHargaProduk() 
    {
        return $this->harga_produk;
    }
}
?>
